js/src/forum/index.js

```diff
@@ -31,26 +32,23 @@ const styles = {
   spoiler: { prefix: '>!', suffix: '!<', blockPrefix: '>! ', multiline: true, trimFirst: true },
 };
 
-const applyStyle = (id) => {
-  // This is a nasty hack that breaks encapsulation of the editor.
-  // In future releases, we'll need to tweak the editor driver interface
-  // to support triggering events like this.
```

Comment:
This comment should remain. The hack isn't that we call `app.composer` instead of a provided composer instance, but rather that we directly access the editor's DOM. 

On a similar note, I would prefer for this function to take the editor driver instance, and then call `styleSelectedText` with `editor.el` so we can keep the encapsulation break in one place.